SparkPost/2web2ui
# EO-802 Add in a date-cutover divider for Health Score V2

### What Changed
 - Added vertical reference line between June 6th and June 7th with label.

### How To Test
 - Change system date to after June 7th,
 - Go to health score dashboard and select a date that encompasses June 7th. Make sure the divider line is there. 
 - Click on a facet to go to heath score page. Make sure the divider line is there too. 

### To Do
- [x] Check with TPM to get design checked off.
